src/order/increase_order_utils.cairo

```diff
@@ -2,12 +2,86 @@
 use starknet::ContractAddress;
 
 // Local imports.
-use satoru::order::{base_order_utils::ExecuteOrderParams, order::Order};
+use satoru::order::{
+    base_order_utils::ExecuteOrderParams, order::{Order, OrderType}, error::OrderError
+};
+use satoru::oracle::{oracle_utils, error};
+use satoru::market::market_utils;
+use satoru::swap::swap_utils;
+use satoru::bank::bank::{IBankDispatcher, IBankDispatcherTrait};
+use satoru::position::{
+    position::PositionTrait, position_utils, position_store_utils, error::PositionError,
+    increase_position_utils
+};
+use satoru::event::event_utils;
+
+// External imports.
+use alexandria_data_structures::array_ext::SpanTraitExt;
 
 // This function should return an EventLogData cause the callback_utils
 // needs it. We need to find a solution for that case.
 #[inline(always)]
-fn process_order(params: ExecuteOrderParams) { //TODO
+fn process_order(params: ExecuteOrderParams) -> event_utils::EventLogData {
```

Comment:
Can you add documentation on this function like it's done with `validate_oracle_block_numbers`

Reply:
Gotcha adding this